bbb-file-upload/deploy.sh

```
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)